cli/src/commands/browse.ts

```diff
@@ -0,0 +1,28 @@
+import Command from '../utils/command';
+import terminalLink from 'terminal-link';
+
+export default class Browse extends Command {
+  public static description = 'list templates';
+  public static hidden = false;
+  public static examples = [
+    `$ ${Command.cliBinName} browse`,
+  ];
+
+  public static args = [];
+
+  public static strict = true;
+
+  public async run() {
+    this.parse(Browse);
+    const templates = await this.lycanClient.listTemplates();
+    if (templates.length === 0) {
+      this.log('Sorry. Templates are not available yet.');
+      return;
+    }
+    for (const { id, name, previewImageUrl, githubUrl} of templates) {
+      const previewImageLink = terminalLink('preview image', previewImageUrl);
+      const githubLink = terminalLink('source', githubUrl);
+      this.log(`${name}: See ${previewImageLink}, get the ${githubLink} from GitHub, or try it with '${Command.cliBinName} try ${id}'`);
```

Comment:
`try` command will be in a future PR?

Also, maybe requires highlighting of a shell command like this: `$ ${Command.cliBinName}`

Reply:
indeed, try is in the works.
the highlighting is with ticks (i.e. `'foo'`) for easy of copy paste. leaving as is.